### ISSUE TITLE
&& inside backquotes  is not rendered

### ISSUE THREAD

**Issue by AkihiroSuda** [cpuguy83/go-md2man#61]
```console
$ cat a.md 
`foo && bar`
$ go-md2man < a.md > a.1
$ cat a.1 
.nh

.PP
\fB\fCfoo \&\& bar\fR
$ man ./a.1 | cat

foo  bar
```

The rendered man page lacks `&&` symbols.

go-md2man version: 1029f53b6507e27158d89cd489669559c1c700a3 (Jul 6, 2020)

**Comment by AkihiroSuda** [comment]
@cpuguy83 PTAL

**Comment by cpuguy83** [comment]
Fixed by #74
